packages/aws-rfdk/lib/deadline/lib/spot-event-plugin-fleet.ts

```diff
@@ -0,0 +1,627 @@
+   *
+   * @default - A new security group will be created automatically.
+   */
+  readonly securityGroups?: ISecurityGroup[];
+
+  /**
+   * User data that instances use when starting up.
+   *
+   * @default - User data will be created automatically.
+   */
+  readonly userData?: UserData;
+
+  /**
+   * The Block devices that will be attached to your workers.
+   *
+   * @default - The default devices of the provided ami will be used.
+   */
+  readonly blockDevices?: BlockDevice[];
+
+  /**
+   * Indicates how to allocate the target Spot Instance capacity
+   * across the Spot Instance pools specified by the Spot Fleet request.
+   *
+   * @default - SpotFleetAllocationStrategy.LOWEST_PRICE.
+   */
+  readonly allocationStrategy?: SpotFleetAllocationStrategy;
+
+  /**
+   * Indicates whether Spot Fleet should replace unhealthy instances.
+   *
+   * @default true
+   */
+  readonly replaceUnhealthyInstances?: boolean;
+
+  /**
+   * Indicates whether running Spot Instances are terminated when the Spot Fleet request expires.
+   *
+   * @default true
+   */
+  readonly terminateInstancesWithExpiration?: boolean;
+
+  /**
+   * Where to place the instance within the VPC.
+   *
+   * @default - Private subnets.
+   */
+  readonly vpcSubnets?: SubnetSelection;
+
+  /**
+   * The end date and time of the request, in UTC format (YYYY -MM -DD T*HH* :MM :SS Z).
+   * After the end date and time, no new Spot Instance requests are placed or able to fulfill the request.
+   *
+   * @default - the Spot Fleet request remains until you cancel it.
+   */
+  readonly validUntil?: Expiration;
+
+  /**
+   * Properties for setting up the Deadline Worker's LogGroup
+   * @default - LogGroup will be created with all properties' default values and a prefix of "/renderfarm/".
+   */
+  readonly logGroupProps?: LogGroupFactoryProps;
+
+  /**
+   * An optional provider of user data commands to be injected at various points during the Worker configuration lifecycle.
+   * You can provide a subclass of InstanceUserDataProvider with the methods overridden as desired.
+   *
+   * @default: Not used.
+   */
+  readonly userDataProvider?: IInstanceUserDataProvider;
+}
+
+/**
+ * Interface for Spot Event Plugin Worker Fleet.
+ */
+export interface ISpotEventPluginFleet extends IConnectable, IScriptHost, IGrantable {
+  /**
+   * Allow access to the Worker's remote command listener port (configured as a part of the
+   * WorkerConfiguration) for an IConnectable that is either in this stack, or in a stack that
+   * depends on this stack. If this stack depends on the other stack, use allowRemoteControlTo().
+   * See https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/remote-control.html
+   *
+   * Common uses are:
+   *
+   *   Adding a SecurityGroup:
+   *     `workerFleet.allowRemoteControlFrom(securityGroup)`
+   *
+   *   Adding a CIDR:
+   *     `workerFleet.allowRemoteControlFrom(Peer.ipv4('10.0.0.0/24').connections)`
```

Comment:
Looking at the implementation of `allowRemoteControlFrom()`, perhaps this shouldn't pass in `.connections`?

Reply:
Oh, so it is
```
export interface IConnectable {
    readonly connections: Connections;
```

and 
```
export declare class Connections implements IConnectable {
    readonly connections: Connections;
```

This means we could have `Peer.ipv4('10.0.0.0/24').connections.connections.connections` and it would still work.
I removed it from here and from my unit-tests as it is indeed redundant. Thanks!